### ISSUE TITLE
customize keybindings

### ISSUE THREAD

**Issue by galantra** [mgsloan/todoist-shortcuts#99]
Hi, I have a German keyboard. Therefore, I can only enter an asterisk * by pressing "Ctrl + +" (control and plus sign). I'd like to change the shortcut. Can I do it anyhow?

**Comment by mgsloan** [comment]
Hi, there is no builtin way to change the shortcuts right now.  This would be a very nice feature!

I documented a manual process for doing this: https://github.com/mgsloan/todoist-shortcuts#how-to-customize-keyboard-shortcuts

**Comment by Wolfsblvt** [comment]
Yes! It sadly overcomplicates keyboard usage for QUERTZ (german) keyboards.

**Comment by galantra** [comment]
I found the process of customizing the add-ons a bit daunting because I use Firefox, not Chrome. So I procrastinated on that task.

The only thing that bothered me about the shortcuts was that I didn't get the asterisk shortcuts to work.

I've noticed that on a US keyboard, the asterisk is not that on a US keyboard, the asterisk is not available without shift on the main keyboard (as I call it). However, I saw that it is available as a single key on the _numpad_ – on both US and QWERTZ keyboards.

If you don't have a numpad, you can still activate CAPSLOCK and then use some of the shortcuts that need an asterisk.

**Comment by GaryFurash** [comment]
Please implement

**Comment by kellyarwine** [comment]
Please implement

**Comment by sonofhypnos** [comment]
I would find this useful!

**Comment by mmcintyre123** [comment]
I would also find this useful!
